libmscore/bracket.h

```diff
@@ -41,6 +41,7 @@ class Bracket final : public Element {
       // horizontal scaling factor for brace symbol. Cannot be equal to magY or depend on h
       // because layout needs width of brace before knowing height of system...
       qreal _magx;
+      Measure* _measure;
```

Comment:
Could you please add an initialization of this variable (`Measure* _measure = nullptr;`) to make its default value defined?